mindsdb_native/libs/phases/type_deductor/type_deductor.py

```diff
@@ -198,8 +202,27 @@ def get_column_data_type(self, data, full_data, col_name):
         else:
             curr_data_type, curr_data_subtype = None, None
 
+        # Check for Tags subtype
+        if curr_data_subtype != DATA_SUBTYPES.ARRAY:
+            lengths = []
+            unique_tokens = set()
+            can_be_tags = True
+            for item in data:
+                try:
+                    item_tags = [t.strip() for t in item.split(',')]
```

Comment:
the `try except` can be replaced with a simple `if` statement:
```
if isinstance(item, str):
    item_tags = [t.strip() for t in item.split(',')]
    lengths.append(len(item_tags))
    unique_tokens = unique_tokens.union(set(item_tags))
else:
    can_be_tags = False
    break
```

Reply:
Noted, simplified this even further.